### DISCUSSION TITLE
Projects (beta) - group by repo

### DISCUSSION THREAD

**Discussion by jesfer** [community/community#5217] [Projects and Issues] [answered]
I want to group my issues by repo. Draft issues would grouped under "No repo" or similar heading as they dont belong to any repo.

**Comment by Totktonada** [comment]
Looks as subtask of https://github.com/github/feedback/discussions/5448. However it is good to have it as an explicit feature request.

**Comment by polevaultweb** [comment]
I am managing projects that include releases from multiple repositories, being able to group a view by repo would be very handy to see exactly what repositories are in scope for the overall release.

**Comment by Totktonada** [comment, marked as answer]
It seems, it is tracked in the https://github.com/github/roadmap/issues/283 roadmap issue now:

> - [ ] Group-by assignee and repos

It is in the Q4 2021 milestone ATM.

**Comment by azenMatt** [comment]
we just released this 😁  

https://github.blog/changelog/2021-12-16-the-new-github-issues-12-16-update/

Check it out and let me know what you think!

**Reply by polevaultweb** [reply]
Excellent, thanks so much @azenMatt.

Would be great to group a Kanban board by repo too 😄
